indico/modules/events/registration/schemas.py

```diff
@@ -0,0 +1,14 @@
+from marshmallow.fields import Integer, String
```

Comment:
missing file header

Reply:
tip, you can use `python bin/maintenance/update_header.py indico` to check (we should probably add this to CI)

Reply:
Agreed, we could add it in CI. Just noticed a test from the room booking passed through.